### PR TITLE
Updated with new search engine

### PR DESCRIPTION
**Pull request by Luci-d**
Spyse.com is in-line with shodan and censys IMO.

P.S. This is the best tools repository. Very thanks.

### CODE REVIEW

**Comment by hackerorya** [comment]
Hacking tools

File: README.md [hunk 1]
```diff
@@ -592,6 +592,7 @@ performance of any of your sites from across the globe.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://censys.io/"><b>Censys</b></a> - platform that helps information security practitioners discover, monitor, and analyze devices.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://www.shodan.io/"><b>Shodan</b></a> - the world's first search engine for Internet-connected devices.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://2000.shodan.io/#/"><b>Shodan 2000</b></a> - do you use Shodan for everyday work? This tool looks for randomly generated data from Shodan.<br>
```
Comment (hackerorya) [inline README.md:594]: My story Im ethical hacker please halpe me global  im leazy  and i need evrything is free  my asisten work for me i want hack any user name and password without any time west so please halpe me make for me evrything easy follow my heart  like magic install me automatic any android siting   i want evry thing without credit card login emale  another i dont have atime  so get start  i want  make new hacking application  like 1 click then  hack any thing so please halpe me world php ruby perl python

**Review by lopezregalodelo-a11y — COMMENTED** [review]
Updated

**Review by lopezregalodelo-a11y — APPROVED** [review]
Update

**Review by lopezregalodelo-a11y — CHANGES_REQUESTED** [review]
Updated

**Pull request merged**